src/main/com/smithsmodding/armory/client/model/block/unbaked/MaterializedBlockModel.java

```diff
@@ -28,12 +26,17 @@
  */
 public class MaterializedBlockModel implements IModel {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)